types/recharts/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Recharts 0.22
+// Type definitions for Recharts 0.23
```

Comment:
There is no version 0.23, you should either use 0.22 or 1.0 according to their websites